src/db/connection.js

```diff
@@ -1,14 +1,43 @@
-const Sequelize = require('sequelize');
-
-const connection = new Sequelize(
-    process.env.PAM_DATABASE,
-    process.env.PAM_DATABASE_USER,
-    process.env.PAM_DATABASE_PASSWORD,
-    {
-        dialect: 'postgres',
-        host: process.env.PAM_DATABASE_URL,
-        port: process.env.PAM_DATABASE_PORT
-    }
-);
-
-module.exports = connection;
+let promise;
+let sequelize = null;
+
+const {PAM_DATABASE, PAM_DATABASE_USER, PAM_DATABASE_PASSWORD, PAM_DATABASE_URL, PAM_DATABASE_PORT} = process.env;
```

Comment:
Add conditional to check if all env vars are in place first before continuing...